pootle/apps/pootle_statistics/models.py

```diff
@@ -551,6 +551,13 @@ def get_similarity(self):
     def is_similarity_taken_from_mt(self):
         return self.submission.similarity < self.submission.mt_similarity
 
+    def get_suggested_words(self):
```

Comment:
hmm... I didn't think about it.
This is a method of `ScoreLog` object and there exists `get_paid_words` method. This object gives us a context where everything is about stats, scores and counters. I like shorter naming `_words` vs. `_wordcount`.